include/mbgl/ios/MGLMapView.h

```diff
@@ -82,6 +82,13 @@ IB_DESIGNABLE
 /** The button shown in the lower-right of the map which when pressed displays the map attribution information. */
 @property (nonatomic, readonly) UIButton *attributionButton;
 
+/** Query the visible map for features at a given point. 
+*
+*   @param point A point on screen, for example at a user gesture.
+*   @param radius A distance in which to query features around the point.
```

Comment:
Should clarify that the distance is measured in screen points, not a physical unit or coordinate delta.